### ISSUE TITLE
Hide Placeholder When Start Typing

### ISSUE THREAD

**Issue by rience** [RobinHerbots/Inputmask#1248]
Hi,

Is it possible to hide placeholder as soon as user put single character?

**Comment by RobinHerbots** [comment]
@rience , 

No, but I will add that.  For the moment you can use the html5 placeholder  and the jitmasking option in the inputmask to accomplish the behavior.
